API.md

```diff
@@ -1,445 +1,648 @@
+            <td>string</td>
+            <td>Default value is us-east-1<br/>
+Location valid values are us-west-1, us-west-2, eu-west-1, eu-central-1, ap-southeast-1, ap-northeast-1, ap-southeast-2, sa-east-1(defaults to us-east-1, optional).</td>
+            </tr>
+               </tbody>
+</table>
+
+
+__Example__
+```py
+try:
+    minioClient.make_bucket("mybucket", location="us-east-1")
+except ResponseError as err:
+    print(err)
+```
+
+---------------------------------------
+<a name="list_buckets">
+#### list_buckets()
+Lists all buckets.
+
+__Parameters__
+
+|Return   |Type   |Description   |
+|---|---|---|
+|``bucketList``   |*function*   |List of all buckets. |
+|``bucket_name``   |*string*   |Bucket name. |
+|``bucket.creation_date`` |*time*   |Time: date when bucket was created. |
+
+__Example__
+```py
+buckets = minioClient.list_buckets()
+for bucket in buckets:
+    print(bucket.name, bucket.creation_date)
+```
+---------------------------------------
+<a name="bucket_exists">
+#### bucket_exists(bucket_name)
+Checks if a bucket exists.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucket_name``   |*string*   |Name of the bucket. |
+
+__Example__
+
+```py
+try:
+    print(minioClient.bucket_exists("mybucket"))
+except ResponseError as err:
+    print(err)
+```
+---------------------------------------
+<a name="remove_bucket">
+#### remove_bucket(bucket_name)
+Removes a bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucket_name``   |*string*   |Name of the bucket. |
+
+__Example__
+```py
+try:
+    minioClient.remove_bucket("mybucket")
+except ResponseError as err:
+    print(err)
+```
+---------------------------------------
+<a name="list_objects">
+#### list_objects(bucket_name, prefix, recursive=False)
+Lists objects in a bucket.
+
+__Parameters__
+
+| Param  |Type  | Description  |
+|---|---|---|
+|``bucket_name``   |*string*   | Name of the bucket.  |
+|``objectPrefix``   | *string*   |The prefix of the objects that should be listed. |
+|``recursive``   | *bool*   |``True`` indicates recursive style listing and ``False`` indicates directory style listing delimited by '/'. Optional default is False.   |
+
+__Return Value__
+
+<table>
+    <thead>
+        <tr>
+            <th>Param</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+           object
+            </td>
+            <td> Object</td>
+            <td> Iterator for all the objects in the bucket, the object is of the format:
+            <ul>
+            <li>object.object_name string: name of the object. </li>
+            <li>object.size int: size of the object.</li>
+            <li>object.etag string: etag of the object. </li>
+            <li>object.last_modified datetime.datetime: modified time stamp. </li>
+            </ul>
+            </td>
+            </tr>
+               </tbody>
+</table>
+

```

Comment:
working on fix so we have same theme for html table as well as markdown with @rushenn